modules/nifti/src/niftireader.cpp

```diff
@@ -162,14 +163,15 @@ const DataFormatBase* niftiDataTypeToInviwoDataFormat(const nifti_image* niftiIm
 
     return DataFormatBase::get(type, components, precision);
 }
-std::shared_ptr<NiftiReader::VolumeSequence> NiftiReader::readData(const std::string& filePath) {
+std::shared_ptr<NiftiReader::VolumeSequence> NiftiReader::readData(std::string_view filePath) {
+    checkExists(filePath);
 
     /* read input dataset, but not data */
-    std::shared_ptr<nifti_image> niftiImage(nifti_image_read(filePath.c_str(), 0),
+    std::shared_ptr<nifti_image> niftiImage(nifti_image_read(SafeCStr{filePath}.c_str(), 0),
```

Comment:
file paths can be longer than 120
```suggestion
    std::shared_ptr<nifti_image> niftiImage(nifti_image_read(SafeCStr<255>{filePath}.c_str(), 0),
```

Reply:
thats fine, 120 is only for the stack